### DISCUSSION TITLE
An abbreviation mark

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#86] [General]
Just from the curiosity:
How would you interpret the abbreviation mark over p and u?
![Latin-breve](https://user-images.githubusercontent.com/8024702/142415183-d9386dcf-b0d5-41d8-9c23-61ff245aeeb6.png)

https://commons.wikimedia.org/wiki/File:Latin-breve.png

**Comment by psb1558** [comment]
It's a variant of the combining zigzag (U+035B). In Gothic script, the zigzag flows into a flourish that connects with the base character. In Junicode this variant (accessible via cv81) looks like this:
<img width="133" alt="image" src="https://user-images.githubusercontent.com/16215767/142477541-25341d45-2dbc-4eaf-8bc8-8afef1db2db3.png">
Note that the zigzag should be present, at least in vestigial form. Some representations leave it off, but this is a mistake. It usually stands for vowel + s or vowel + r or r + vowel or s + vowel: above, ræ and er.

**Reply by psb1558** [reply]
Junicode also has variant forms that connect correctly with letters of various shapes, e.g.
<img width="76" alt="image" src="https://user-images.githubusercontent.com/16215767/142478220-4ade5e34-af78-4e60-86af-1b953823d508.png">

**Comment by jsbien** [comment]
Thanks!
